sys/mips/include/setjmp.h

```diff
@@ -40,6 +40,11 @@
 #include <sys/cdefs.h>
 
 #define	_JBLEN	95		/* size, in longs (or long longs), of a jmp_buf */
+#ifdef __mips_n32
```

Comment:
Looks like mips/_types.h has a suitable typedef for register_t so I'd say use that instead?
```
#if defined(__mips_n64) || defined(__mips_n32)
typedef __int64_t      __register_t;
```